lib/spans/streaming-span-event.js

```diff
@@ -197,8 +197,19 @@ class StreamingHttpSpanEvent extends StreamingSpanEvent {
       agentAttributes.url = null
     }
 
+    if (agentAttributes.host) {
+      this.addAgentAttribute('server.address', agentAttributes.host)
+      agentAttributes.host = null
```

Comment:
I noticed the nullifying of these properties isn't doing what we expect. I will file another ticket to deal with that